package.json

```diff
@@ -38,6 +38,10 @@
     "test": "jest && jest-it-up",
     "test:watch": "jest --watch"
   },
+  "resolutions": {
+    "ink": "3.1.0",
+    "react-reconciler": "0.24.0"
+  },
```

Comment:
why are these being forced?

Reply:
`ink` 3.2.0 has peer deps for `"react": ">=16.8.0"` and only devDeps for react 17. So it should be fine??

I see why `react-reconciler` must be in the resolutions though

Reply:
Doesn't `"react-reconciler": "0.24.0"` already force the version of `react-reconcile` in `ink`? No big deal, just trying to understand if I had a wrong assumption of how the resolution works in yarn. Going to approve